### ISSUE TITLE
IDE0079 false positive while operators usage on properties

### ISSUE THREAD

**Issue by slavanap** [dotnet/roslyn#53913]
**Version Used**: 
.NET 5.0
VisualStudio.16.Release/16.10.0+31321.278

**Steps to Reproduce**:

```csharp
using System;

namespace ConsoleApp1 {
    class Program {
        static int MyProp {
            get => 1;
            set => Console.WriteLine("Hello world x{0}!", value);
        }

        static void Main() {
            MyProp++;
            //Console.WriteLine(MyProp); // uncommenting this line removes the IDE0079 notice
        }
    }
}
```

**Expected Behavior**:

No IDE0079, property getter and setter are utilized via operator++.

**Actual Behavior**:

`IDE0052  Private property 'Program.MyProp' can be converted to a method as its get accessor is never invoked.`
is shown

**Comment by Youssef1313** [comment]
Duplicate of https://github.com/dotnet/roslyn/issues/43191?

**Comment by slavanap** [comment]
@Youssef1313, yes. I've provided additional scope of cases there. Thanks.
